core/src/test/java/org/mskcc/oncotree/utils/TumorTypesUtilTest.java

```diff
@@ -72,22 +81,20 @@ private boolean testValuesMatch(String value1, String value2) {
 
     @Test
     public void testFullOncoTreeCodeListByCode() throws Exception {
-        Set<TumorType> tumorTypesAsPreparedForTumorTypesEndpoint = simulateTumorTypeApiResponse();
-        HashMap<String,TumorType> returnedTumorTypeMap = new HashMap<>(tumorTypesAsPreparedForTumorTypesEndpoint.size());
-        for (TumorType tumorType : tumorTypesAsPreparedForTumorTypesEndpoint) {
-            returnedTumorTypeMap.put(tumorType.getCode(), tumorType);
-        }
         int failureCount = 0;
         StringBuilder failureReport  = new StringBuilder();
+        Map<String,TumorType> returnedTumorTypeMap = getReturnedFullTumorTypeMap();
```

Comment:
This one is not a `HashMap` specifically. Is this because it is not what is returned?